src/main/webapp/js/finscholar/homepagecontroller/homepagecontroller.js

```diff
@@ -16,54 +16,122 @@
 
 goog.module('finscholar.homepagecontroller');
 
-const JsactionActionFlow = goog.require('jsaction.ActionFlow');
-const JsactionDispatcher = goog.require('jsaction.Dispatcher');
-const JsactionEventContract = goog.require('jsaction.EventContract');
-const GoogDom = goog.require('goog.dom');
-const GoogSoy = goog.require('goog.soy');
+const googDom = goog.require('goog.dom');
+const googSoy = goog.require('goog.soy');
+const jsactionActionFlow = goog.require('jsaction.ActionFlow');
+const jsactionDispatcher = goog.require('jsaction.Dispatcher');
+const jsactionEventContract = goog.require('jsaction.EventContract');
 const {homepage} = goog.require('finscholar.homepagecontroller.templates');
```

Comment:
Sorting. Uppercase comes before lowercase.

Reply:
get it.